modules/creating-a-custom-training-image.adoc

```diff
@@ -0,0 +1,118 @@
+:_module-type: PROCEDURE
+
+[id='creating-a-custom-training-image_{context}']
+= Creating a custom training image
+
+You can create a custom training image by adding packages to a base training image.
+
+.Prerequisites
+
+* You have chosen a training image to use as the base for your custom image. 
+ifndef::upstream[]
++
+See link:https://access.redhat.com/articles/rhoai-supported-configs[Supported Configurations] for a list of the {productname-short} base training images and their preinstalled packages.
+endif::[]
+
+* You can access the base training image.
+* You have Podman installed in your local environment, and you can access a container registry.
++
+For more information about Podman and container registries, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/building_running_and_managing_containers/index[Building, running, and managing containers].
+
+
+.Procedure
+
+. In a terminal window, create a directory for your work, and change to that directory. 
++
+In the example commands in this section, `my_training_image` is the name of the working directory _and_ the name of the image you create.
```

Comment:
suggest that you make it clear whether using the same name for the working directory and the custom image is recommended? If is not recommended/best practice - consider using different names in the example

Reply:
Removed the reference to the name of the working directory, because the user can choose any name.